### DISCUSSION TITLE
Offer

### DISCUSSION THREAD

**Discussion by moreauftheobald** [Bodmer/TFT_eSPI#1779] [New processor/display support or capability request]
Hi Bodmer,

Thanks a lot for your very good library, I want to make an offer to you, because i need some changes on the library, but i know that it is not easy for you to do the work.

What i need:
- ESP32-S3 support
- 16 bit wiring support (SSD1963 LCD)
- 8 bit colors mode support (SSD1963 LCD)

i know that at this time you did'nt own an ESP32-S3 chips, so here is my offer (on two point):

-1 I own 2 ESP32 -S3 devkit V1, so i can easely send you one (the same for SSD1963 Screen). 
-2 I'm not good enought in C coding to do by myself the change i need, but i can easely test code and do minor change, so if you want i can test myself some change you could do and try to resolve parts of the problem.

I got many TFT (ST7796 SPI and 8bit, ILI9486 SPI, ST7789, and GC9A01, and some other small display in i2C), so i can test the change on many screen to be sure that the change does not affect the other dysplay.

thank for reading me.

Regards

F. MOREAU

**Comment by Bodmer** [comment]
Thank you for the offer. Unfortunately I have to decline as it would take too much of my time. Eventually I will add support for the S3 but it will have to wait at least 6 months. 24 bit colour support will have to wait too.

**Comment by moreauftheobald** [comment]
Hi,
I think i have excuse myself, you should maybe have misunterstood my previous message. When i said i need esp32-S3, and other i dont mean i need it immédiatly, it could be next year, i dont mind ;)

Anyway i can understand your opinion about my offer.

By the way, my offer of sending you one off my ESP32-S3 is still open, if you need it for your work, that's not a problem for me to send it to you. for the moment i could not use it as Arduino IDE framework is still not ready, and i'm working on my proto with ESP32.

like this you'll be able to wok on this processor and send it back to me when you will not need it anymore.

regards.
